Resources/public/js/views/services/plugins/ez-savedraftplugin.js

```diff
@@ -160,11 +189,14 @@ YUI.add('ez-savedraftplugin', function (Y) {
                 contentType: service.get('contentType'),
                 parentLocation: service.get('parentLocation'),
                 fields: fields,
-            }, Y.bind(function (error, response) {
-                if ( !error ) {
+            }, Y.bind( function (error, response) {
+                if ( error ) {
```

Comment:
that's weird way of building the code, since there already a check against `error` in `_saveDraftCallback`, this is where you should decide if `serverSideErrorCallback` should be called or not